turbomoleio/output/parser.py

```diff
@@ -37,6 +37,7 @@
 float_number_d_re = r"[+-]?[0-9]*[.]?[0-9]+D[+-]\d{2}"
 float_number_e_re = r"[+-]?[0-9]*[.]?[0-9]+E[+-]\d{2}"
 float_number_all_re = r"[+-]?[0-9]*[.]?[0-9]+([ED][+-]\d{2})*"
+float_number_all_re2 = r"[+-]?[0-9]*[.]?[0-9]+(?:[ED][+-]\d{2})?"
```

Comment:
couldn't this be a replacement for `float_number_all_re`? or does this break some other parsing?

Reply:
Indeed, I have removed float_number_all_re2 and just used float_number_all_re.